underlay/src/main/resources/config/datamapping/aouSR2019q4r4/entity/conditionConcept/all.sql

```diff
@@ -0,0 +1,10 @@
+SELECT
```

Comment:
Added for the JobSequencerTest which is using the aouSR2019q4r4_verily indexer. Chose this for test because it's not being used by anyone that I know of.